src/table/table.stories.ts

```diff
@@ -0,0 +1,268 @@
+		if (model.header[index].sorted) {
+			// if already sorted flip sorting direction
+			model.header[index].ascending = model.header[index].descending;
+		}
+		model.sort(index);
+	}
+}
+
+
+class CustomHeaderItem extends TableHeaderItem {
+	// used for custom sorting
+	compare(one: TableItem, two: TableItem) {
+		const stringOne = (one.data.name || one.data.surname || one.data).toLowerCase();
+		const stringTwo = (two.data.name || two.data.surname || two.data).toLowerCase();
+
+		if (stringOne > stringTwo) {
+			return 1;
+		} else if (stringOne < stringTwo) {
+			return -1;
+		} else {
+			return 0;
+		}
+	}
+}
+
+
+const simpleModel = new TableModel();
+simpleModel.data = [
+	[new TableItem({data: "Name 1"}), new TableItem({data: "qwer"})],
+	[new TableItem({data: "Name 3"}), new TableItem({data: "zwer"})],
+	[new TableItem({data: "Name 2"}), new TableItem({data: "swer"})],
+	[new TableItem({data: "Name 4"}), new TableItem({data: "twer"})]
+];
+simpleModel.header = [
+	new TableHeaderItem({data: "Name"}), new TableHeaderItem({data: "hwer", style: {"width": "auto"} })
+];
+
+function sort(model, index: number) {
+	if (model.header[index].sorted) {
+		// if already sorted flip sorting direction
+		model.header[index].ascending = model.header[index].descending;
+	}
+	model.sort(index);
+}
+
+function simpleSort(index: number) {
+	sort(simpleModel, index);
+}
+
+
+storiesOf("Table", module).addDecorator(
+		moduleMetadata({
+			imports: [
+				NFormsModule,
+				TableModule,
+				TranslateModule.forRoot()
+			],
+			declarations: [
+				DynamicTableStory,
+				ExpansionTableStory
+			]
+		})
+	)
+	.addDecorator(withKnobs)
+	.add("default", () => ({
+		template: `
+		<ibm-table
+			[model]="model"
+			[size]="size"
+			[showSelectionColumn]="showSelectionColumn"
+			[striped]="striped"
+			(sort)="simpleSort($event)">
+		</ibm-table>
+	`,
+		props: {
+			model: simpleModel,
+			simpleSort: simpleSort,
+			size: selectV2("size", {Small: "sm", Normal: "md", Large: "lg"}, "md", "table-size-selection"),
+			showSelectionColumn: boolean("showSelectionColumn", true),
+			striped: boolean("striped", true)
+		}
+	}))
+	.add("with expansion", () => ({
+		template: `
+			<app-expansion-table
+				[size]="size"
+				[showSelectionColumn]="showSelectionColumn"
+				[striped]="striped">
+			</app-expansion-table>
+		`,
+		props: {
+			size: selectV2("size", {Small: "sm", Normal: "md", Large: "lg"}, "md", "table-size-selection"),
+			showSelectionColumn: boolean("showSelectionColumn", true),
+			striped: boolean("striped", true)
+		}
+	}))
+	.add("with dynamic content", () => ({
+		template: `
+			<app-custom-table
+				[size]="size"
+				[showSelectionColumn]="showSelectionColumn"
+				[striped]="striped">
+			</app-custom-table>
+		`,
+		props: {
+			size: selectV2("size", {Small: "sm", Normal: "md", Large: "lg"}, "md", "table-size-selection"),
+			showSelectionColumn: boolean("showSelectionColumn", true),
+			striped: boolean("striped", true)
+		}
+	}));
+
```

Comment:
Can also do this to distinguish each story more:
![image](https://user-images.githubusercontent.com/14989804/44595157-3c18ee80-a796-11e8-99f9-379e2fa992a9.png)

Reply:
Can, shouldn't in this case. This follows what other stories in this repo do.

We can consider refactoring all of them.